src/helper/certificate.ts

```diff
@@ -133,17 +133,33 @@ export async function generateCertificate(
     "font.fnt"
   );
   console.log(fontPath);
+  const xAlignment = event.certificate?.alignmentX;
+  const yAlignment = event.certificate?.alignmentY;
+
+  if(!xAlignment || !yAlignment) throw { success:false, message:"ALIGN-PARAMS NOT FOUND" }
```

Comment:
Put a check here if the yAlignemnt and xAlighment are one of "LEFT", "RIGHT", "CENTER".